packages/web/src/pages/sign-up-page/components/AccountHeader.tsx

```diff
@@ -0,0 +1,87 @@
+import { Avatar, Box, Flex, Text } from '@audius/harmony'
+
+import {
+  getCoverPhotoField,
+  getHandleField,
+  getNameField,
+  getProfileImageField
+} from 'common/store/pages/signon/selectors'
+import { useMedia } from 'hooks/useMedia'
+import { useSelector } from 'utils/reducer'
+
+export const AccountHeader = () => {
+  const coverPhoto = useSelector(getCoverPhotoField)
+  const profileImage = useSelector(getProfileImageField)
+  const { value: displayName } = useSelector(getNameField)
+  const { value: handle } = useSelector(getHandleField)
+
+  const { isMobile } = useMedia()
+
+  const avatarSize = isMobile ? 72 : 120
+
+  return (
+    <Box>
+      <Box h={isMobile ? 96 : 168} css={{ overflow: 'hidden' }}>
+        <Box
+          h='100%'
+          w='100%'
+          border='default'
+          css={{
+            backgroundImage: `linear-gradient(0deg, rgba(0, 0, 0, 0.20) 0%, rgba(0, 0, 0, 0.00) 100%), url(${coverPhoto.url}), url(${coverPhoto.url})`,
```

Comment:
do you know what this linear gradient does?

Reply:
honestly not sure lol

Reply:
we should check if it does nothing. I assume the intent is just to use the background image?

Reply:
that's what i think too, cause the mobile version doesnt have this blur effect for example

Reply:
@szonana for context

Reply:
The linear gradient helps ensure contrast when white photos are uploaded!  
<img width="901" alt="image" src="https://github.com/AudiusProject/audius-protocol/assets/116679440/86cbf0ba-92d2-4aed-9f2b-676c2138d3fe">

Reply:
Thank you so much for the context!

Reply:
> that's what i think too, cause the mobile version doesnt have this blur effect for example

@dylanjeffers it should! 🙏 
![image](https://github.com/AudiusProject/audius-protocol/assets/116679440/bf465c38-b7de-4de4-ba34-ef61fe411d92)